### ISSUE TITLE
eden/examples/stylegan.ipynb Encoding section latent_representations not generated by notebook

### ISSUE THREAD

**Issue by jagatfx** [abraham-ai/eden#1]
When running the example notebook `eden/examples/stylegan.ipynb`, the latent_representations for the various examples used in the Encoding section are not generated. 

```
locations = [
    os.path.join("../external/stylegan-encoder", 'latent_representations/starrynight.npy'),
    os.path.join("../external/stylegan-encoder", 'latent_representations/monetlandscape.npy'),
    os.path.join("../external/stylegan-encoder", 'latent_representations/fields-of-gold-jane-small.npy'),
    os.path.join("../external/stylegan-encoder", 'latent_representations/monalisa1024.npy'),
    os.path.join("../external/stylegan-encoder", 'latent_representations/girlpearl.npy'),
    os.path.join("../external/stylegan-encoder", 'latent_representations/frida.npy'),
    os.path.join("../external/stylegan-encoder", 'latent_representations/keefe.npy'),
    os.path.join("../external/stylegan-encoder", 'latent_representations/scream.npy')
]
```

This results in error:
`FileNotFoundError: [Errno 2] No such file or directory: './external/stylegan-encoder/latent_representations/starrynight.npy'`

It is unclear how to generate the necessary files.

**Comment by genekogan** [comment]
The way they are generated is by first encoding an image into the latent space. This hasn't been added to the notebook yet (it just says "Instructions for encoding TBD") but is planned. For now, you can see how to do this from the [original repository](https://github.com/pbaylies/stylegan-encoder/)

**Comment by jagatfx** [comment]
@genekogan Are you accepting pull requests to resolve TBD items and places where the notebooks fail as is? Also, is highlighting something like this helpful or would you prefer a different process?

**Comment by genekogan** [comment]
Yes, absolutely! And yes, the issue tracker is best for keeping track of bugs, and low-level development issues, whereas the [forum](http://forum.abraham.ai) or [discord](https://discordapp.com/channels/573691888050241543/573691888482123778) is better for high-level planning/strategy/discussion. I am on a tiny holiday next week so I may not get to any PRs until early September though.
